tests/resources/Util.robot

```diff
@@ -59,53 +59,39 @@ Run VIC Machine Command
     [Return]  ${output}
 
 Cleanup VIC Appliance On Test Server
+    [Tags]  secret
     Run Keyword And Ignore Error  Gather Logs From Test Server
-    # Let's attempt to cleanup any container related to the VCH appliance first
-    ${list}=  Run  govc ls /ha-datacenter/vm
-    ${list}=  Split To Lines  ${list}
-    :FOR  ${vm}  IN  @{list}
-    \   Continue For Loop If  '${vm}'=='/ha-datacenter/vm/${vch-name}'
-    \   ${raw}=  Run  govc vm.info -json=true ${vm}
-    \   ${status}  ${message}=  Run Keyword And Ignore Error  Should Contain  ${raw}  ${vch-name}
-    \   ${name}=  Run Keyword If  '${status}'=='PASS'  Run  govc vm.info -json\=true ${vm} | jq -r '.VirtualMachines[0].Name'
-    \   ${uuid}=  Run Keyword If  '${status}'=='PASS'  Run  govc vm.info -json\=true ${vm} | jq -r '.VirtualMachines[0].Config.Uuid'
-    \   Run Keyword If  '${status}'=='PASS'  Run  govc vm.destroy ${name}
-    \   Run Keyword If  '${status}'=='PASS'  Run  govc datastore.rm ${uuid}
-
-    # Then we can try to cleanup the VCH itself
-    ${uuid}=  Run  govc vm.info -json\=true ${vch-name} | jq -r '.VirtualMachines[0].Config.Uuid'
-    ${output}=  Run  govc vm.destroy ${vch-name}
-    ${output}=  Run  govc pool.destroy %{GOVC_RESOURCE_POOL}/${vch-name}
-    ${output}=  Run  govc datastore.rm ${vch-name}
+    ${output}=  Run  bin/vic-machine-linux delete --name=${vch-name} --target=%{TEST_URL} --user=%{TEST_USERNAME} --password=%{TEST_PASSWORD} --force=true --compute-resource=%{TEST_RESOURCE} --timeout %{TEST_TIMEOUT}
+    Run Keyword And Ignore Error  Should Contain  ${output}  Completed successfully
     ${output}=  Run  rm -f ${vch-name}-*.pem
-    ${output}=  Run  govc datastore.rm VIC/${uuid}
+    [Return]  ${output}
 
 Cleanup Datastore On Test Server
```

Comment:
Biggest issue is that vSAN datastore lists two IDs for each folder:

office2-sfo2-dhcp167:devbox bcorrie$ govc datastore.ls | wc -l
      36
office2-sfo2-dhcp167:devbox bcorrie$ govc datastore.rm 081a0e3a3b963c51c630283eeea688218b27ea2b21b21f6b565b118d9dd83f04
office2-sfo2-dhcp167:devbox bcorrie$ govc datastore.ls | wc -l
      34